### ISSUE TITLE
[Spike] Best practices for items that should be ignored by screen readers

### ISSUE THREAD

**Issue by eliserichards** [mozilla-mobile/fenix#19426]
Some view items are not useful when navigating with a screen reader, and we do not necessarily want to add content descriptions that would make navigation frustrating. 

## Deliverables

- Are there best practices in the accessibility community for handling this? Document these practices and file issues for fixing in our app.

**Comment by priyanka0906** [comment]
@eliserichards  we can set the value of the Content Description to null. In  this way it is not read by the screen reader.
I have read it here : https://www.essentialaccessibility.com/blog/two-issues-one-solution-for-android-accessibility

**Comment by MReschenberg** [comment]
I reached out to the rest of the a11y team to see if anyone had more to offer here, but we all agree the above suggestion is the way to go :)
